app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/ReaderFragment.kt

```diff
@@ -141,6 +209,9 @@ class ReaderFragment : CoreReaderFragment() {
     }
   }
 
+  private fun getSharedPrefSettings() =
+    activity?.getSharedPreferences(SharedPreferenceUtil.PREF_KIWIX_MOBILE, 0)
+
   override fun hasValidFileAndUrl(url: String?, zimFileReader: ZimFileReader?) =
     super.hasValidFileAndUrl(url, zimFileReader) && url != HOME_URL
```

Comment:
I'd like to see HOME_URL totally deleted, I believe it being used is no longer correct